assets/js/index.js

```diff
@@ -0,0 +1,310 @@
+  nextAnswer.setAttribute("id", "answer-container");
+  answerSection.append(nextAnswer);
+  displayQuestions();
+};
+
+// when questions finished - display form to enter initials
+const gameOver = () => {
+  clearTimeout(timing);
+  document.querySelector('input[name="initials"]').value = "";
+  gameOverScreen.classList.remove("hide");
+  questionScreen.classList.add("hide");
+  header.classList.add("hide");
+  const userScoreText = document.getElementById("display-final-score");
+  userScoreText.innerHTML = "Your final score is: " + score;
+};
+
+// submit user input and sort high score list
+const onFormSubmit = (event) => {
+  event.preventDefault();
+  const formInput = document.querySelector('input[name="initials"]');
+  const userInitials = formInput.value;
+  const newScore = {
+    score: score,
+    initials: userInitials,
+  };
+  highScoreArray.push(newScore);
+  highScoreArray.sort((a, b) => b.score - a.score);
+  highScoreArray.splice(5);
+  localStorage.setItem("highScores", JSON.stringify(highScoreArray));
+
+  gameOverScreen.classList.add("hide");
+  header.classList.add("hide");
+  highScoreScreen.classList.remove("hide");
+
+  displayHighScores();
+};
+
+// return to beginning screen
+const goHome = () => {
+  clearAnswers();
+  highScoreScreen.classList.add("hide");
+  header.classList.remove("hide");
+  titleScreen.classList.remove("hide");
+  timerItem.classList.add("hide");
+  questionNumber = 0;
+  score = 0;
+  timerStarted = false;
+  timer = timerAmount;
+};
+
+// if high scores button is pressed, display high scores screen
+const viewHighScores = () => {
+  clearTimeout(timing);
+  header.classList.add("hide");
+  if (!titleScreen.classList.contains("hide")) {
+    titleScreen.classList.add("hide");
+  } else if (!questionScreen.classList.contains("hide")) {
+    questionScreen.classList.add("hide");
+    clearAnswers();
+  }
+  highScoreScreen.classList.remove("hide");
+  displayHighScores();
+};
+
+// clear answers
+const clearAnswers = () => {
+  const currentAnswers = document.getElementById("answer-container");
+  while (currentAnswers.firstChild) {
+    currentAnswers.removeChild(currentAnswers.firstChild);
+  }
+};
+
+// display high scores from local storage
+const displayHighScores = () => {
+  const scoreList = clearScores();
+
+  if (highScoreArray.length === 0) {
+    highScoreError.classList.remove("hide");
+  } else {
+    if (!highScoreError.classList.contains("hide")) {
+      highScoreError.classList.add("hide");
+    }
+    for (let i = 0; i < highScoreArray.length; i++) {
+      const scoreItem = document.createElement("li");
+      scoreItem.textContent =
+        highScoreArray[i].initials.toUpperCase() +
+        " - " +
+        highScoreArray[i].score;
+      scoreList.appendChild(scoreItem);
+    }
+  }
+};
+
+// clear scores page
+const clearScores = () => {
+  const scoreList = document.getElementById("score-list");
+  while (scoreList.firstChild) {
```

Comment:
This could be done by just removing the element from the DOM using the `scoreList.remove()`